dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -254,10 +250,7 @@ void Microsoft::Windows::AppNotifications::Helpers::RegisterAssets(std::wstring
 
     if (displayName.length() == 0)
     {
-        wil::unique_cotaskmem_string processName;
-        THROW_IF_FAILED(GetCurrentProcessPath(processName));
-
-        displayName = processName.get();
+        THROW_IF_FAILED(wil::GetModuleFileNameExW(GetCurrentProcess(), nullptr, displayName));
```

Comment:
Should this be a wil construct? Not wstring?

Reply:
Looks like GetModuleFileName just takes in a generic string_type. I can change this to wil::unique_cotaskmem_string which is the default string_type used.